kore/src/main/scala/org/kframework/unparser/Unparse.scala

```diff
@@ -74,37 +80,37 @@ object ToKast {
    * @param prec The current precedence level
    * @param k The term to print
    */
-  def unparse(b:StringBuilder, inParen: Boolean, prec: Int, k: K): Unit = k match {
-    case KToken(s, sort) => b ++= "#token(\"" + escape(s) + "\",\"" + escape(sort.name) + "\")"
-    case InjectedKLabel(l) => b ++= "#klabel("+apply(l)+")"
-    case KVariable(v) => b ++= v.toString
-    case KApply(l, List()) => b ++= unparse(inParen,l)+"(.KList)"
+  def unparse(b:String=>Unit, inParen: Boolean, prec: Int, k: K): Unit = k match {
```

Comment:
`b` is not particularly revealing name (it made sense when in the previous version when it was StringBuilder)
if I understand correctly, it accumulates string either in a StringBuilder or in output stream, so having a more telling name would help readability